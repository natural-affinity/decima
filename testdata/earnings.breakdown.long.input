decima --breakdown 750.25
