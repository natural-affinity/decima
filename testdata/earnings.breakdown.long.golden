You owe: $75.03

Takehome: $675.23
Earnings: $750.25
 Percent: 10.00%
